packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -1,19 +1,26 @@
-import { getCollectionHooks } from '../mutationCallbacks';
-import { Revisions, ChangeMetrics } from '../../lib/collections/revisions/collection'
-import { extractVersionsFromSemver } from '../../lib/editor/utils'
-import { ensureIndex } from '../../lib/collectionIndexUtils'
-import { htmlToPingbacks } from '../pingbacks';
-import { diff } from '../vendor/node-htmldiff/htmldiff';
-import { editableCollections, editableCollectionsFields, editableCollectionsFieldOptions, sealEditableFields, MakeEditableOptions } from '../../lib/editor/make_editable';
-import { getCollection } from '../../lib/vulcan-lib/getCollection';
-import { CallbackHook } from '../../lib/vulcan-lib/callbacks';
-import { createMutator } from '../vulcan-lib/mutators';
-import * as _ from 'underscore';
-import cheerio from 'cheerio';
-import { onStartup } from '../../lib/executionEnvironment';
-import { dataToHTML, dataToWordCount } from './conversionUtils';
-import { Globals } from '../../lib/vulcan-lib/config';
+import {getCollectionHooks} from '../mutationCallbacks'
+import {ChangeMetrics, Revisions} from '../../lib/collections/revisions/collection'
+import {extractVersionsFromSemver} from '../../lib/editor/utils'
+import {ensureIndex} from '../../lib/collectionIndexUtils'
+import {htmlToPingbacks} from '../pingbacks'
+import {diff} from '../vendor/node-htmldiff/htmldiff'
+import {
```

Comment:
Note: I think @Discordius (Oliver Habryka) actually prefers the one-line rather than 5-line version of this. (Oli's less involved with the codebase now, and I'm not sure what other people think. I used to go around changing the codebase in the way you're doing here and Oli trained it out of me when he was the primary decisionmaker). 

His reasoning was something like "this line isn't actually that important, and having it take up more vertical space assigns the line disproportionate importance, and meanwhile parsing out the different variables isn't useful enough to counteract that".

There's a few other places you do this, I'm not personally gonna ask you to change them, but noting it's at least not considered by everyone to be a strict improvement.

Reply:
This is not exactly an intentional change (though it matches my preferences) - my editor does the  automatically when it auto-imports things for you/etc.  I mostly tried to avoid reformatting code I didn't touch,  but didn't make effort to undo things like this